docs/src/content/docs/community/showcase/xensql.md

```
@@ -0,0 +1,28 @@
+
+![XenSQL Screenshot](../../../../assets/showcase-images/xensql-1.png)
+<table>
+  <tr>
+    <td align="center"><img src="../../../../assets/showcase-images/xensql-2.png?raw=true" width="100%"></td>
+    <td align="center"><img src="../../../../assets/showcase-images/xensql-3.png?raw=true" width="100%"></td>
+  </tr>
+</table>
+
+**[XenSQL](https://github.com/Bare7a/XenSQL)** is a **fast, local-first SQL desktop workbench** built with **Go, Wails and React**. It brings together PostgreSQL, MySQL/MariaDB, and SQLite in one clean, native-feeling interface - with zero cloud, zero telemetry, and zero accounts.
+
+### Key Highlights
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)